src/platform/platform_linux.c

```diff
@@ -199,67 +199,6 @@ CxPlatFree(
 #endif
 }
 
-void
-CxPlatPoolInitialize(
-    _In_ BOOLEAN IsPaged,
-    _In_ uint32_t Size,
-    _In_ uint32_t Tag,
-    _Inout_ CXPLAT_POOL* Pool
-    )
-{
-#ifdef CX_PLATFORM_DISPATCH_TABLE
-    PlatDispatch->PoolInitialize(IsPaged, Size, Pool);
```

Comment:
Looks like we don't use these Pool* dispatch table functions anymore. Can you remove them too please?